sale_order_qty_change_no_recompute/tests/test_sale_order_qty_change.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests.common import Form, SavepointCase
+
+
+class TestSaleOrderQtyChange(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.product = cls.env["product.product"].create(
+            {"name": "Test Product", "list_price": 25.00}
+        )
+        sale_form = Form(cls.env["sale.order"])
+        sale_form.partner_id = cls.env.ref("base.res_partner_12")
+        with sale_form.order_line.new() as cls.line_form:
+            cls._set_init_line(cls)
+
+    def _set_init_line(self):
```

Comment:
Why doing this sub-method? In fact the only thing you should test is:

- Fix a price on the line
- Change product > check the price changes
- Change quantity > check that the price doesn't change.